src/context/ComputerBoardContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, createContext, useEffect } from 'react';
+import axios from 'axios';
+
+export interface ComputerBoardInterface {
+    name: String,
+    grid: Array<any>,
+    shipList: Array<any>,
+    size: Number,
+    everyShipSunk: boolean
+}
+
+const defaultBoard = {
+  name: 'default',
+  grid: [],
+  shipList: [],
+  size: 0,
+  everyShipSunk: false,
+};
+export const ComputerBoardContext = createContext <ComputerBoardInterface | null>(null);
+
+export const ComputerBoardContextProvider = function (props : any) {
+  const [computerBoard, setComputerBoard] = useState(defaultBoard);
+
+  useEffect(() => {
+    const apiURL = 'http://localhost:8080/boards';
+    const fetchBoards = async () => {
```

Comment:
We call this apiURL here and then in the PlayerBoardContext.  Suppose we make a spelling error with the url?  Can you set this api to a variable in, for example, a `api.tsx` file, and then import this variable in both files?

Reply:
Done.